### DISCUSSION TITLE
Bug with text editor

### DISCUSSION THREAD

**Discussion by 1kvalitet1** [devlinx9/muon-ssh#113] [General]
Hi, you have a good program, but one bug prevents me from using it.
When editing a file using a text editor, for example (nano or vim)
Scrolling down the file, the cursor changes only the last line, after editing the file it becomes random, because letters are eaten, lines are confused and it is not possible to read scrolling down the file.

screen
https://github.com/user-attachments/assets/124a024e-2161-4133-a2cd-31695f790ba9
https://github.com/user-attachments/assets/43e301c7-729a-4d2d-af84-cdd44356b36e

**Comment by devlinx9** [comment]
Can you check the new version: 2.3.0?
In that version set the terminal type to ansi: Settings -> Terminal -> Terminal Type -> ansi
please let me know if it works for you
